### PR TITLE
Adding platforms and channel doc section

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Adds a simple section on docs to talk about supported platforms and channel

This PR is targeting master as this is just documentation and is already valid for the current version, I will update develop and v1 once this is merged.

### CODE REVIEW

File: doc/platforms.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Supported platforms
+
+Flame runs on top of Flutter, so its supported platforms depended on how mature/stable that platform is supported by Flutter.
```
Comment (spydon) [inline doc/platforms.md:3]: Depended -> depend
That platform is supported by -> support for each platform is by

File: doc/platforms.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+At the moment, mobile has full support, and we try to keep supporting web the best we can.
+
+## Flutter channels
+
+Flame keeps it support on the stable and beta channel, dev and master channel should work, but we don't support them, this mean that issues happening outside stable or beta channel, are not a priority.
```
Comment (spydon) [inline doc/platforms.md:9]: Split this up into more sentences (too many commas).

File: doc/platforms.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Flame web
+
+To use Flame on web you need to be on the beta channel of Flutter and the game should be ran using skia rendering, otherwise weird side effects may happen.
```
Comment (spydon) [inline doc/platforms.md:13]: Ran -> running

File: doc/platforms.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+To run your game using skia in release mode, use the following command:
+
+`$ flutter run -d Chrome --release --dart-define=FLUTTER_WEB_USE_SKIA=true`
+
+To run with skia on debug mode, just start the project using the conventional command:
+
+`$ flutter run -d Chrome`
+
+And well the app starts and the debug is setup, press the `k` key. that will enable the `CanvasKit` mode.
```
Comment (spydon) [inline doc/platforms.md:23]: Well -> when

Comment (spydon) [inline doc/platforms.md:23]: . that -> , that
Or capital T

File: doc/platforms.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Supported platforms
```
Comment (luanpotter) [inline doc/platforms.md:1]: nit: titles in Title Case

**Comment by erickzanardo** [comment]
@spydon @luanpotter @renancaraujo I have fixed our lint, apparently the flutter format command now printed some info about collecting usage data, and that was breaking our script, I changed so we check on the exit code instead of checking if the command had printed anything.

**Pull request merged**